pyproject.toml

```diff
@@ -55,6 +55,8 @@ pandas = "^2.1.4"
 pillow = "^10.3.0"
 pyyaml = ">=5.1,<7.0.0"
 typing-extensions = "^4.12.2"
+transformers = "^4.46.3"
+semchunk = "^2.2.0"
```

Comment:
we might need to move these to extras.
for example, if `datasets` would use `docling-core`, they would not accept this dependency.

Reply:
These have now been moved to an extra.